### PR TITLE
Added my comments. A pull request == a conversation :')

### PR DESCRIPTION
**Pull request by AriSweedler**
Yo! I'm so happy that you shared your dotfiles with me LOL. I learned a HEALTHY amount going over them. Normally, I find gems of widom online at a very slow rate, but HERE this repo was, with EVERYTHING I wanted!! Lit! You da man.

### CODE REVIEW

**Comment by AriSweedler** [comment]
Uhhh I mostly just used this pull request to annotate your code with my comments/thoughts, but I made like 1 or 2 changes. It doesn't matter if you discard all the changes, but I hope you at least find the commends interesting!

File: .bash_profile [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#oooooooh so clean
```
Comment (caziz) [inline .bash_profile:1]: I'll be removing .macos here because it slowed login way down

File: .gitconfig [hunk 2]
```diff
@@ -8,28 +8,53 @@
 	email = caziz@ucla.edu
 [core]
 	excludesfile = /Users/christopheraziz/.gitignore_global
+	# I know I said something in class, but. Loooool xD I respect this.
```
Comment (caziz) [inline .gitconfig:11]: send me your vim book!

File: .gitconfig [hunk 3]
```diff
@@ -8,28 +8,53 @@
...
 	editor = subl -n -w
 
+# I changed these spaces to tabs. It doesn't matter, but I did it anyway. Tbh I
+# don't know what I'm making these chagnes for, I'm only submitting this pull
+# request so you can see m y commends on your code in an organized fashion, but
+# I guess you can merge some hunks if you want? Lol idk how to merge only
+# specific hunks. But anyway, I'm all about advanced usage of git (and GitHub)
+# - I wonder what a GUI-less pull request is supposed to look like?? Anyway,
+# lit colors :')
```
Comment (caziz) [inline .gitconfig:20]: I found recently that I like using tabs but having the IDE insert spaces but that might be just a fad. However, it looks like I've been inconsistent which is super embarrassing.

File: .bash_aliases [hunk 4]
```diff
@@ -1,4 +1,8 @@
 # CS 136 DeterLab
+# Check out ssh config files!! They're pretty sweet. I don't know how to do the
+# tunnel with an ssh config file (yet) but I'm 100% sure that there's a way.
+# That being said, this is simpler (I had to special case .local/bin and
+# .ssh/config in my update (bootstrap) script
```
Comment (caziz) [inline .bash_aliases:5]: I added an SSH config! it also does tunneling :)

File: .bash_aliases [hunk 5]
```diff
@@ -56,12 +71,22 @@ mk() {
 }
 mksh() { if [ "$#" -ne 1 ]; then echo "usage: mksh file"; else mk "$1" '#!/usr/bin/env bash'; fi; }
 mkpy() { if [ "$#" -ne 1 ]; then echo "usage: mkpy file"; else mk "$1" '#!/usr/bin/env python'; fi; }
+# I like these^ :D I use vim abbreviations to do it, typeing '#!b<SPACEBAR>'
```
Comment (caziz) [inline .bash_aliases:74]: I honestly don't use these enough for them to be justified. I like your strategies better :/

File: .bash_aliases [hunk 6]
```diff
@@ -125,6 +151,9 @@ ghcl() {
 }
 
 # macOS commands
+# nice lol. You can also use <CTRL-COMMAND-q>, but this alias is definitly way
+# cooler. Where did you learn all these macos hacks!? I wanna read whatever you
+# read. Do you  have a recommendation?
```
Comment (caziz) [inline .bash_aliases:156]: I like yours better!

Comment (caziz) [reply .bash_aliases:156]: YES! this is the holy grail: https://github.com/mathiasbynens/dotfiles

Comment (caziz) [reply .bash_aliases:156]: I stole plenty of my stuff from that repo ;)

File: .gitconfig [hunk 7]
```diff
@@ -39,32 +64,38 @@
...
-	--pretty=format:'%C(bold yellow)%d\
-	%C(reset)%C(cyan)%h %C(dim white)-\
-	%C(reset)%s %C(dim blue)(%cr)\
-	%C(dim white)<%an>%C(reset)' \
-	--abbrev-commit --date=short
+	# If you want this in one line, then you can predefine "git-pretty"
+	# formattings! Check it out.
+	# Also, I didn't know the difference between author and committer until
+	# just now. Nice! Imma use authoer instead, like you. I think that's
+	# much better
```
Comment (caziz) [inline .gitconfig:71]: I also had no idea there was a difference

File: .gitconfig [hunk 8]
```diff
@@ -39,32 +64,38 @@
...
 	st = status -sb
 	lg = log --oneline
-	gr = log --graph \
-	--pretty=format:'%C(bold yellow)%d\
-	%C(reset)%C(cyan)%h %C(dim white)-\
-	%C(reset)%s %C(dim blue)(%cr)\
-	%C(dim white)<%an>%C(reset)' \
-	--abbrev-commit --date=short
+	# If you want this in one line, then you can predefine "git-pretty"
+	# formattings! Check it out.
```
Comment (caziz) [inline .gitconfig:68]: I didn't know! I will

File: .gitconfig [hunk 9]
```diff
@@ -39,32 +64,38 @@
...
+	# Also, I didn't know the difference between author and committer until
+	# just now. Nice! Imma use authoer instead, like you. I think that's
+	# much better
+	# (https://stackoverflow.com/questions/18750808/difference-between-author-and-committer-in-git)
+	gr = log --graph --pretty=chrislog --abbrev-commit --date=short
 	graph = !git gr --all
 
 	ci = commit
 	acm = !git add . && git commit -m
+	# "git acm" is functionally identical to "git cam" (right?)
```
Comment (caziz) [inline .gitconfig:78]: Almost. The -a option stages files that have been modified and deleted, but new files you have not told Git about are not affected. (not that I can usually keep subtle differences like this straight)

File: .gitconfig [hunk 10]
```diff
@@ -39,32 +64,38 @@
...
 	cm = commit -m
 	cam = commit -am
 	amend = commit --amend
 	cane = commit --ammend --no-edit
 
 	branches = branch -a
+	# looool this is alias is troll
```
Comment (caziz) [inline .gitconfig:85]: I rarely use these, but I got them after reading some rants about git and then landing on this:
https://gggritso.com/human-git-aliases

File: bootstrap.sh [hunk 11]
```diff
@@ -2,7 +2,15 @@
 
 # use -i for interactive mode and -f to force
 
-FILES=`git ls-tree -r master --name-only`
+# git ls-files does the same thing!
+# Also, you might wanna replace backticks with $(). It's the same, but people
+  # (online people, cuz no one is pedantic enough to care IRL... right? LOL) people
+  # somehow tell me that $() is better (http://mywiki.wooledge.org/BashFAQ/082)
+    # However, I see you wrote this in sh not bash (for portability?) Idk lol.
```
Comment (caziz) [inline bootstrap.sh:9]: lol no I'm just bad

File: bootstrap.sh [hunk 12]
```diff
@@ -27,9 +37,14 @@ for file in $FILES; do
...
         fi
-        read -p "> " confirm && [[ $confirm == [yY] || $confirm == [yY][eE][sS] ]] || continue
+        # The "-n" option for `read` will limit user input to 1 character. I
+        # like the feel of this better
+        read -p "> " -n 1 confirm && [[ $confirm == [yY] ]] || continue
     fi
     if [[ $FORCE = true || $INTERACTIVE = true ]]; then
+        # the rm spooks me. I'm stupid, so I'd get burned if I left this as is
+        # on my machine lol. However, not just rm'ing it makes it more
+        # complicated so I also respect the simplicity.
```
Comment (caziz) [inline bootstrap.sh:47]: I figured explicitly adding an option to running the script  and/or typing 'yes'  is enough for me to think about it

File: bootstrap.sh [hunk 13]
```diff
@@ -27,9 +37,14 @@ for file in $FILES; do
             echo "~/"$file" does not exist."
             echo "Create link to "$file"? [y/n]"
         fi
-        read -p "> " confirm && [[ $confirm == [yY] || $confirm == [yY][eE][sS] ]] || continue
+        # The "-n" option for `read` will limit user input to 1 character. I
+        # like the feel of this better
```
Comment (caziz) [inline bootstrap.sh:41]: righto!

File: .bash_aliases (hunk 4 above)
Comment (caziz) [reply .bash_aliases:5]: with what I have now, putting the config in dotfiles/.ssh seems to copy the path into home just fine without a special case!

File: .gitconfig [hunk 14]
```diff
@@ -8,28 +8,53 @@
...
+	new = green bold
+	whitespace = normal red
 [color "status"]
-  added = green
-  changed = yellow
-  untracked = red
+	added = green
+	changed = yellow
+	untracked = red
+[pretty]
```
Comment (caziz) [inline .gitconfig:37]: this is cool!

**Comment by AriSweedler** [comment]
Hope you been well ahaha. I just noticed that this was open

**Pull request closed without merging**